packages/foreman/foreman/foreman.spec

```diff
@@ -450,8 +450,9 @@ Meta package to install requirements for Rackspace compute resource support.
 Summary: Foreman VMware support
 Group:  Applications/System
 # start specfile vmware Requires
-Requires: %{?scl_prefix}rubygem(fog-vsphere) >= 3.2.1
```

Comment:
https://github.com/theforeman/foreman/pull/7213 should clear this up. I'll fix it manually now.